examples/webpack.config.local.js

```diff
@@ -33,7 +33,7 @@ function makeLocalDevConfig(EXAMPLE_DIR = LIB_DIR, linkToLuma) {
     '@luma.gl/webgl-state-tracker': `${ROOT_DIR}/node_modules/@luma.gl/webgl-state-tracker`,
     '@luma.gl/webgl2-polyfill': `${ROOT_DIR}/node_modules/@luma.gl/webgl2-polyfill`,
     '@luma.gl/shadertools': `${ROOT_DIR}/node_modules/@luma.gl/shadertools`,
-    '@luma.gl/addons': `${ROOT_DIR}/node_modules/@luma.gl/addons`
+    '@loaders.gl/core': `${ROOT_DIR}/node_modules/@loaders.gl/core`
```

Comment:
Why is this needed?

Reply:
To ensure everyone is using the same loaders.gl/core.